index.js

```diff
@@ -66,8 +74,6 @@ class PD {
       server: true,
     });
     await discovery.flushed();
-    const foundPeers = this.store.findingPeers();
-    foundPeers();
```

Comment:
this means nothing so killed it, either you wait for flush to call found peers or not but mostly a demo / testing thing